tor_core/helpers.py

```diff
@@ -79,23 +80,23 @@ def clean_list(items):
     return cleaned
 
 
-def send_to_slack(message, config):
+def send_to_slack(message, config, channel='#general'):
```

Comment:
Just came in to comment, this is a beautiful way of retaining backwards compatibility despite a lot of added functionality. Very nice job!